prime-router/build.gradle.kts

```diff
@@ -393,9 +393,8 @@ tasks.register<JavaExec>("primeCLI") {
 
     // Use arguments passed by another task in the project.extra["cliArgs"] property.
     doFirst {
-        val cliArgs = project.extra["cliArgs"]
```

Comment:
I goofed and didn't test this right.  In the new gradle, this will throw an error without first checking that it exists.